src/database/models/organizationExtension.js

```diff
@@ -39,6 +39,7 @@ module.exports = (sequelize, DataTypes) => {
 			},
 			mentee_visibility_policy: { type: DataTypes.STRING },
 			external_mentee_visibility_policy: { type: DataTypes.STRING },
+			organization_name: { type: DataTypes.STRING },
```

Comment:
@rakeshSgr rename it to just name please, Also update the corresponding DB diagrams for documentation as well.